{"uuid": "5ca078be-4501-4216-9188-f98bf6e2bb22", "children": ["92764491-2d1e-44d1-b743-d6240ae65aef"], "befores": [{"name": "setup", "status": "failed", "statusDetails": {"message": "AssertionError: User icon is not presented, probably unauthorised user\n", "trace": "  File \"/Users/gremlin/PycharmProjects/stepik_lessons/venv/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/gremlin/PycharmProjects/stepik_lessons/venv/lib/python3.8/site-packages/_pytest/fixtures.py\", line 918, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/gremlin/PycharmProjects/stepik_lessons/venv/lib/python3.8/site-packages/_pytest/fixtures.py\", line 774, in call_fixture_func\n    res = next(it)\n  File \"/Users/gremlin/PycharmProjects/stepik_lessons/module_5/test_product_page.py\", line 106, in setup\n    page.should_be_authorized_user()\n  File \"/Users/gremlin/PycharmProjects/stepik_lessons/module_5/pages/base_page.py\", line 28, in should_be_authorized_user\n    assert self.is_element_present(*BasePageLocators.USER_ICON), \\\n"}, "start": 1608037075253, "stop": 1608037091670}], "start": 1608037075253, "stop": 1608037091682}